odoo/tools/mail.py

```diff
@@ -295,6 +297,20 @@ def html_keep_url(text):
     return final
 
 
+def get_html_content(html):
```

Comment:
html2unformattedtext?

Reply:
Mmmmh, but it is not returning valid html. get_html_content from "html" source, somehow I think I will get a valid html content, a parsed html content, ... here it is basically text with some entities. Not sure about the naming, but would update it ^^